### PR TITLE
Dashboard config

### PR DESCRIPTION
**Pull request by fm1psy**
This PR closes issue #35

### CODE REVIEW

File: dashboard/main_page.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import streamlit as st
```
Comment (FarihaChoudhury) [inline dashboard/main_page.py:2]: missing docstrings

File: dashboard/main_page.py
Comment (FarihaChoudhury) [inline dashboard/main_page.py:1]: don't forget to do a readme!

File: dashboard/.streamlit/config.toml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+[logger]
```
Comment (Zhi-704) [inline dashboard/.streamlit/config.toml:1]: Should probably mention that this is the configuration file for the dashboard here

File: dashboard/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+pylint
+pytest
+streamlit
+altair
+boto3
+python-dotenv
+psycopg2-binary
+st-pages
+pandas
```
Comment (Zhi-704) [inline dashboard/requirements.txt:9]: Do you use pandas yet?

File: dashboard/.streamlit/config.toml (hunk 2 above)
Comment (fm1psy) [reply dashboard/.streamlit/config.toml:1]: This is a configuration file so I don't think comments are needed

File: dashboard/main_page.py (hunk 1 above)
Comment (fm1psy) [reply dashboard/main_page.py:2]: Not anymore

File: dashboard/requirements.txt (hunk 3 above)
Comment (fm1psy) [reply dashboard/requirements.txt:9]: Not yet, but based on previous work I expect to use it.

File: dashboard/main_page.py
Comment (fm1psy) [reply dashboard/main_page.py:1]: I will do not worry

**Review by Zhi-704 — APPROVED** [review]
Looks good!

**Pull request merged**